### ISSUE TITLE
Re-upload stuff

### ISSUE THREAD

**Issue by LexterS999** [daycat/stupid-simple-pingtunnel#1]
Please, if you have original files from author's releases, re-upload it on your git. Author deleted all stuff from releases.

**Comment by daycat** [comment]
hmm, that's weird. I'll rebuild the files and then change the script in this case

**Comment by LexterS999** [comment]
Cool. So you have zip archives from last release?

**Comment by daycat** [comment]
Yes, do you want these?

**Comment by LexterS999** [comment]
Yes, that would be nice. Can you share the link? And dont forget to fix your auto installation script.
